src/main/java/com/afornalik/ox/model/player/Player.java

```diff
@@ -0,0 +1,80 @@
+package com.afornalik.ox.model.player;
+
+import com.afornalik.ox.model.board.FieldStatus;
+
+public class Player {
```

Comment:
Your player has state and exposes it to other classes. It do not have any other functionality. Maybe instead of using getters in other places in the code ask player to do something with this information (e.g. you could provide board as a parameter and ask player to put their symbol on it; or maybe instead of giving score and name separately, you could ask player to construct appropriet message from it and return this message only?)

Reply:
I treat this class like an entity - so it generally only hold player state.

Reply:
Something like a pojo